src/Routes.php

```diff
@@ -24,7 +24,30 @@ class Routes extends AbstractEndpoint {
 	 * @return array|\WP_Error
 	 */
 	public function endpoint_callback( \WP_REST_Request $request ) {
-		$date = [];
+		$data = [];
+
+		// Create a route for each page.
+		$pages_query = new \WP_Query([
+			'post_type' => 'page',
+		]);
+
+		while ( $pages_query->have_posts() ) {
+			$pages_query->the_post();
+
+			$page = get_post();
+
+			$data[] = [
+				'state' => $page->post_name,
+				'url' => str_replace( site_url(), '', get_permalink( $page ) ),
```

Comment:
Can we put `site_url()` outside of the loop since is always going to  be the same value and there is no need to call it `n` times instead of 1